snippets/csharp/VS_Snippets_WebNet/WebParts_ToolZone_Overview/CS/Toolzone_overview.cs

```diff
@@ -9,7 +9,7 @@ public partial class Toolzone_overview_cs : System.Web.UI.Page
 {
   protected void Button1_Click(Object sender, EventArgs e)
   {
-    if(TextBox1.Text == String.Empty | TextBox1.Text.Length < 0)
```

Comment:
What's the hack?

Reply:
I'm at a loss as to what this was supposed to be. A quick search seems to indicate that it's not used at all.

I'll accept this PR with this change.